tools/integrations/manage.sh

```diff
@@ -69,6 +69,12 @@ cmd_publish() {
   echo "Publishing new version ($versioned_image)"
   docker push "$versioned_image"
   docker push "$latest_image"
+
+  # publish spec to cache. do so, by running get spec locally and then pushing it to gcs.
+  local tmp_spec_file; tmp_spec_file=$(mktemp)
+  docker run --rm "$versioned_image" spec | jq .spec > "$tmp_spec_file"
```

Comment:
@sherifnada this will break if spec returns anything but a single airbyte message of type spec. do you have any sense of how bad of an assumption that is. i guess we can protect against this by getting a little fancier with jq.

Reply:
I recommend filtering to the right spec message. It shouldn't be that much harder to filter to only spec messages right? (and there should definitely only be one of those)

Reply:
cool. added new jq logic. it handles:
1. filtering non valid json
2. picking the first spec if multiple are returned
3. failing if no specs are returned